docs/source/features/sheet_management/dataframe.rst

```diff
@@ -4,3 +4,123 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the columns Protex schema defines and the data resides in the rows and columns.
+
+*********************
+ Construct DataFrame
```

Comment:
`Initialize  a Dataframe`. The following descriptions all start with `Initialize`.